src/pages/ProductList/ProductList.js

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import './ProductList.scss';
 
 function ProductList() {
-  return <p>Hi</p>;
+  const [productInfoList, setProductInfoList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/productList.json', { method: 'GET' })
```

Comment:
```suggestion
    fetch('/data/productList.json')
```
단순 데이터 받아오는 요청시에는 GET 메소드를 넣지 않아도 됩니다!